node_test/rpc_test.go

```diff
@@ -13,9 +13,10 @@ import (
 	"github.com/google/go-cmp/cmp"
 	"github.com/rs/zerolog"
 	"github.com/statechannels/go-nitro/channel/state/outcome"
-	"github.com/statechannels/go-nitro/internal/infra"
+	infraRpc "github.com/statechannels/go-nitro/internal/rpc"
```

Comment:
Nit: you have used the alias `interRpc` elsewhere.

Reply:
Good call. Updated in c1ac24e5.